pkgs/applications/misc/bikeshed/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+# build inputs
+, aiofiles
+, aiohttp
+, attrs
+, certifi
+, cssselect
+, html5lib
+, isodate
+, json-home-client
+, lxml
+, pillow
+, pygments
+, requests
+, result
+, setuptools
+, tenacity
+, widlparser
+}:
+
+buildPythonApplication rec {
+  pname = "bikeshed";
+  version = "3.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-eIr6AxBgX/cArO9qEYG5jnSiaDIddYiDbAYNqjcnrNc=";
+  };
+
+  # Relax requirements from "==" to ">="
+  # https://github.com/tabatkins/bikeshed/issues/2178
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "==" ">="
+  '';
+
+  propagatedBuildInputs = [
+    aiofiles
+    aiohttp
+    attrs
+    certifi
+    cssselect
+    html5lib
+    isodate
+    json-home-client
+    lxml
+    pillow
+    pygments
+    requests
+    result
+    setuptools
+    tenacity
+    widlparser
+  ];
+
```

Comment:
Package has tests we can run and a pythonImportsCheck can also be added.

Reply:
For some reason, it's not able to run any tests (either here or the dependencies):
```
       > ============================= test session starts ==============================
       > platform linux -- Python 3.9.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
       > rootdir: /build/source
       > collected 0 items                                                              
       >
```

That's when I add `pytestCheckHook` to `checkInputs`.

Reply:
Can you try
```nix
checkPhase = ''
  $out/bin/bikeshed test
'';
```
?

https://github.com/tabatkins/bikeshed/blob/main/.github/workflows/ci.yml#L36-L37